calculation-service/src/test/java/org/java/mentorship/calculation/CalculationApplicationTest.java

```diff
@@ -0,0 +1,14 @@
+package org.java.mentorship.calculation;
+
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class CalculationApplicationTest {
```

Comment:
I think you can delete this test
Application classes don't require coverage anymore